code/game/mecha/mecha.dm

```diff
@@ -508,9 +509,16 @@
 				break
 		if(result)
 			move_result = mechstep(direction)
+
 	//Turning
-	else if(src.dir != direction)
-		move_result = mechturn(direction)
+
+	else if(src.dir!=direction && strafing)

```

Comment:
`else if(src.dir!=direction && strafing)`
Should still be
`else if(src.dir != direction)`
The first && strafing means it will skip over, and not allow it to turn, even when not strafing.

Reply:
@Mechoid Thanks, that should be corrected now.